packages/field-plugin/helpers/vite/src/manifest.ts

```diff
@@ -0,0 +1,30 @@
+import { existsSync, readFileSync } from 'fs'
+import { resolve } from 'path'
+
+export const MANIFEST_FILE_NAME = 'field-plugin.config.json'
+export const MANIFEST_FULL_PATH = resolve(process.cwd(), MANIFEST_FILE_NAME)
+
+export type ManifestOption = {
+  name: string
+  value: string
+}
+
+export type Manifest = {
+  options: ManifestOption[]
+}
+
+export const fileExists = (): boolean => {
+  return existsSync(MANIFEST_FULL_PATH)
+}
+
+export const load = (): Manifest => {
+  try {
+    const content: string = readFileSync(MANIFEST_FULL_PATH, {
+      encoding: 'utf-8',
```

Comment:
You mean without making use of other libraries such as `cosmiconfig`, right?

Reply:
Oh I meant "utf-8" here

Reply:
I've tried without informing the encoding as suggested, @eunjae-lee, but when it's not informed, the `readFileSync` returns a buffer instead of a string. As I'd need to convert it into a string in order to call `JSON.parse`, I think the easiest way would be to keep the encoding part but in its shortcut way:

`const content: string = readFileSync(MANIFEST_FULL_PATH, 'utf-8')`

I'll commit this way and if you may have another option, let me know, please.